ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```diff
@@ -4,6 +4,44 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+function createIntallJunit() {
+  if test -f "${SHARED_DIR}/install-status.txt"
+  then
+    EXIT_CODE=`cat ${SHARED_DIR}/install-status.txt | awk '{print $1}'`
+    INSTALL_STATUS=`cat ${SHARED_DIR}/install-status.txt | awk '{print $2}'`
+    if [ "$EXIT_CODE" ==  0  ]
+    then
+      cat >"${ARTIFACT_DIR}/junit_install.xml" <<EOF
+      <testsuite name="cluster install" tests="2" failures="0">
+        <testcase name="cluster install should succeed"/>
+        <testcase name="cluster bootstrap should succeed"/>
```

Comment:
Apologies for an absolute nitpick, but my brain would love to see the bootstrap result before the install result given it's a precursor.